src/main/java/net/ccbluex/liquidbounce/features/module/modules/world/Scaffold.kt

```diff
@@ -360,6 +362,12 @@ object Scaffold : Module("Scaffold", ModuleCategory.WORLD, Keyboard.KEY_I) {
         }
     }
 
+    @EventTarget
+    fun onSneakSlowDown(event: SneakSlowDownEvent) {
+        event.forward = eagleSpeed
+        event.strafe = eagleSpeed
+    }
+
```

Comment:
Good that you changed the Event below, but now this will not be the same anymore. It has to be * 5 or change the value range to like 1F to 5F, so it makes more sense.